oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/ApplicationConfig.java

```diff
@@ -138,6 +139,15 @@ public static int getISPort() {
         return value;
     }
 
+    public static String getPostLogoutRedirectUri() {
+
+        String value = getProperties().getProperty(POST_LOGOUT_REDIRECT_URI);
```

Comment:
Change the variable name value->postRedirectURI

Reply:
Fixed.